test/inductor/test_cudacodecache.py

```diff
@@ -0,0 +1,89 @@
+# Owner(s): ["module: inductor"]
+
+import ctypes
+
+import torch
+
+from torch._inductor.codecache import AsyncCompile, CUDACodeCache
+from torch._inductor.codegen.cuda.cuda_env import nvcc_exist
+from torch._inductor.exc import CUDACompileError
+from torch.testing._internal.common_utils import TestCase as TorchTestCase
+
+_SOURCE_CODE = r"""
```

Comment:
I feel it's okay since this file is only for `test_cudacodecache`, and the variable follows the underscore prefix convention to mark it as internal only.